src/main/java/com/iexec/worker/feign/CustomCoreFeignClient.java

```diff
@@ -129,21 +129,23 @@ public Optional<ReplicateTaskSummary> getAvailableReplicateTaskSummary(long last
     }
 
     public TaskNotificationType updateReplicateStatus(String chainTaskId, ReplicateStatusUpdate replicateStatusUpdate) {
-        Map<String, Object> arguments = new HashMap<>();
-        arguments.put(JWTOKEN, loginService.getToken());
-        arguments.put("chainTaskId", chainTaskId);
-        arguments.put("statusUpdate", replicateStatusUpdate);
-
-        HttpCall<TaskNotificationType> httpCall = args ->
-                coreClient.updateReplicateStatus((String) args.get(JWTOKEN), (String) args.get("chainTaskId"),
-                        (ReplicateStatusUpdate) args.get("statusUpdate"));
-
-        ResponseEntity<TaskNotificationType> response = makeHttpCall(httpCall, arguments, "updateReplicateStatus");
-        if (!is2xxSuccess(response)) {
+        try {
+            final ResponseEntity<TaskNotificationType> response = coreClient.updateReplicateStatus(
+                    loginService.getToken(),
+                    chainTaskId,
+                    replicateStatusUpdate
+            );
+            final TaskNotificationType notificationType = response.getBody();
+            log.info(replicateStatusUpdate.getStatus().toString() + " [chainTaskId:{}]", chainTaskId);
```

Comment:
```suggestion
            log.info("Updated replicate status to {} [chainTaskId:{}]", replicateStatusUpdate.getStatus().toString(), chainTaskId);
```
(or anything else better, just to contextualize a bit)

Reply:
Please see https://github.com/iExecBlockchainComputing/iexec-worker/pull/499/commits/e1a7dd6e7fd3077a50f1ca56f79a614e2fab629e.